src/components/Filter/util/filterUtil.ts

```diff
@@ -52,11 +52,12 @@ export const getFilterAppliedCount = (filters: FilterSet, filterValues: FilterVa
  * Given a list (Array or Set) of string, return a query string representation.
  * Default behavior is that when all items are selected, nothing is returned.
  */
-export const getUrlFromList = (name: string, list: string[] | Set<string>, count: number) => {
+
+export const getUrlFromList = (name: string, list: string[] | Set<string>, count: number, initialValue: boolean = false) => {
```

Comment:
nit: I recommend renaming the last parameter to something like `hasInitialValue` or something. The way it is now seems a little misleading.